client/src/js/otus/components/__tests__/List.test.js

```diff
@@ -0,0 +1,97 @@
+import { mapStateToProps, mapDispatchToProps, OTUsList } from "../List";
+
+describe("<OTUsList />", () => {
+    let props;
+
+    beforeEach(() => {
+        props = {
+            refId: "foo",
+            term: "bar",
+            verified: true,
+            onLoadNextPage: jest.fn(),
+            documents: ["foo"],
+            page: 1,
+            page_count: 1,
+            renderRow: true,
+
+            references: { detail: { id: 1 } },
+            otus: { verified: true }
+        };
+    });
+
+    it("shoudl render", () => {
+        const wrapper = shallow(<OTUsList {...props} />);
+        expect(wrapper).toMatchSnapshot();
+    });
+
+    it("should renter when [this.props.documents === null]", () => {
+        props.documents = null;
+        const wrapper = shallow(<OTUsList {...props} />);
+        expect(wrapper).toMatchSnapshot();
+    });
+
+    it("should render when [this.props.documents.length == false]", () => {
+        props.documents = false;
+        const wrapper = shallow(<OTUsList {...props} />);
+        expect(wrapper).toMatchSnapshot();
+    });
+
+    it("should return componentDidMount in props", () => {
+        const page = "foo";
+        expect(props.onLoadNextPage).not.toHaveBeenCalled();
+        const wrapper = shallow(<OTUsList {...props} />);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['wrapper' is assigned a value but never used. (no-unused-vars)](https://app.codacy.com/app/virtool/virtool/pullRequest?prid=4564139)